src/main/java/com/codeborne/selenide/Selenide.java

```diff
@@ -101,6 +101,28 @@ public static void open(String relativeOrAbsoluteUrl, String domain, String logi
     mockModalDialogs();
   }
 
+  /**
+   *
+   * The main starting point in your tests.
+   * <p>
+   * Open browser and pass authentication using build-in proxy.
+   * <p>
+   * A common authenticationType is "Basic". See Web HTTP reference for other types.
+   * <p>
+   * {@code Configuration.fileDownload == Configuration.FileDownloadMode.PROXY;} - is required configuration.
```

Comment:
I would rephrase it to something like "this method can only work if `{@code Configuration.fileDownload == Configuration.FileDownloadMode.PROXY;}`"

Reply:
@asolntsev Understood. Thanks!